Dangerfile

```diff
@@ -6,9 +6,23 @@ if (github.pr_body + github.pr_title).include?("WIP")
   warn("Pull Request is Work in Progress")
 end
 
-# Contributors should always provide a changelog when submitting a PR
+# Contributors should always provide a summary when submitting a PR
```

Comment:
We don't currently run danger unfortunately due to issues when people force-push. I don't know if that has been resolved yet 😢